src/azure-cli/azure/cli/command_modules/storage/_client_factory.py

```diff
@@ -306,6 +301,30 @@ def cf_adls_service(cli_ctx, kwargs):
     return t_adls_service(account_url=account_url, credential=credential, **client_kwargs)
 
 
+def cf_file_service(cli_ctx, kwargs):
```

Comment:
This has been added in #22215 and it's now duplicated with https://github.com/Azure/azure-cli/blob/818a0aa958203131d1fc9a43ecda75bbf5adf8f3/src/azure-cli/azure/cli/command_modules/storage/_client_factory.py#L381-L397

Reply:
Thank you. I did not notice that, we use a different function name by coincidence, haha.